test/dotnet-new.IntegrationTests/DotnetNewTestTemplatesTests.cs

```diff
@@ -66,7 +66,7 @@ static DotnetNewTestTemplatesTests()
                 .WithCustomHive(CreateTemporaryFolder(folderName: "Home"))
                 .WithWorkingDirectory(CreateTemporaryFolder())
                 .Execute()
-                .Should().ExitWith(0);
+                .Should().Pass();
```

Comment:
Nit: to match style of the other calls

```suggestion
                .Should()
                .Pass();
```

Please apply to the others